src/main/cli/cli.c

```diff
@@ -5599,7 +5598,6 @@ static void printConfig(char *cmdline, bool doDiff)
             dumpAllValues(HARDWARE_VALUE, dumpMask, "master");
         } else {
             dumpAllValues(MASTER_VALUE, dumpMask, "master");
-
```

Comment:
You should undo this blank line removals that I assume are leftover from earlier code. Then there will be no changes to this file and it can be dropped from the commit.

Reply:
Ok.